docs/migrate.rst

```diff
@@ -1,11 +1,21 @@
 Migration Guide
 ===============
 
+Migrating to Rally 2.9.0
+------------------------
+
+``build_flavor`` becomes a global variable in track templates
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Rally now treats ``build_flavor`` as a :ref:`global variable<advanced_extensions>` in Jinja2 templates. The value of the variable depends on the build flavor reported by Elasticsearch cluster targetted by Rally.
+The ``build_flavor`` becomes a reserved name so user-supplied track parameters must be named differently. Rename your track parameters before upgrade if needed.
```

Comment:
does it make sense to add an error message user will get if they accidentally are using `build_flavor`, in case they search for it, they get here?

Reply:
This wouldn't hurt. Added in https://github.com/elastic/rally/pull/1750/commits/29351b463d84f4bc51f556e6b2454dbdb8983b05.